### PR TITLE
Added dimensions specs for alert triggers

### PR DESCRIPTION
**Pull request by cristianciutea**

### CODE REVIEW

File: src/coscale/api/alert.go [hunk 1]
```diff
@@ -267,3 +278,75 @@ func ParseHandle(handle string) (string, error) {
 	jsonHandle, err := json.Marshal(result)
 	return string(jsonHandle), err
 }
+
+// ParseDimensionSpecs is used to parse the dimensions specs for a metric.
+func ParseDimensionSpecs(format string) (string, error) {
```
Comment (fryckbos) [inline src/coscale/api/alert.go:283]: Can we use the same format for the dimension specs as we use for retrieving data ?

File: src/coscale/command/alert.go [hunk 2]
```diff
@@ -541,9 +546,13 @@ Optional:
 	--servergroupid
 		The servergroup id for which the alert will be triggered.
 	Note: if no server or servergroup is provided the tigger will be set for entire application.
+	--dimensions
+		The dimension specifications for the metric monitored in the following format:
+			<dimension_id1>:[agregator]:<dimension_value_id1,dimension_value_id2...>;...
+		e.g: '1:AVG:1,2,3;2:MAX:*;3:2,4'
```
Comment (fryckbos) [inline src/coscale/command/alert.go]: Format as in `coscale-cli data get`:
```
		e.g.: --dimensionsSpecs='[[1,"AVG(*)"]]'
		      --dimensionsSpecs='[[2,"*"]]'
		      --dimensionsSpecs='[[3,"11,12,13"],[4,"21,22,23"]]'
```

**Review by fryckbos — COMMENTED** [review]
I think we should use an uniform format for the dimension specs in the CLI.

**Review by kidk — APPROVED** [review]

**Comment by kidk** [comment]
@cristianciutea Does this now match the return values from API? At first sight it seems so, but just want to check to make sure. I'm going to send you a script through Slack, could you try it out?

**Review by fryckbos — APPROVED** [review]
Looks good to me !

**Pull request merged**